### PR TITLE
feat(off-policy): support off-policy lag

### PR DESCRIPTION
**Pull request by Gaiejj**
## Description

feat(off-policy): support off-policy lag

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [x] I have read the [CONTRIBUTION](https://omnisafe.readthedocs.io/en/latest/developer/contributing.html) guide. (**required**)
- [x] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [x] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format`. (**required**)
- [x] I have checked the code using `make lint`. (**required**)
- [x] I have ensured `make test` pass. (**required**)

### CODE REVIEW

File: omnisafe/algorithms/off_policy/ddpg_lag.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Copyright 2022-2023 OmniSafe Team. All Rights Reserved.
```
Comment (zmsn-2077) [inline omnisafe/algorithms/off_policy/ddpg_lag.py:1]: 2023

**Review by zmsn-2077 — APPROVED** [review]
LGTM.

File: omnisafe/adapter/offpolicy_adapter.py [hunk 2]
```diff
@@ -62,6 +62,14 @@ def __init__(  # pylint: disable=too-many-arguments
         seed: int,
         cfgs: Config,
     ) -> None:
+        """Initialize the off-policy adapter.
+
+        Args:
+            env_id: The environment id.
```
Comment (muchvo) [inline omnisafe/adapter/offpolicy_adapter.py:68]: how about considering type of parameters?

File: omnisafe/adapter/online_adapter.py [hunk 3]
```diff
@@ -60,25 +60,39 @@ def __init__(  # pylint: disable=too-many-arguments
...
+        OmniSafe provides a set of adapters to adapt the environment to the framework.
+
+        - OnPolicyAdapter: Adapt the environment to the on-policy framework.
+        - OffPolicyAdapter: Adapt the environment to the off-policy framework.
+        - SauteAdapter: Adapt the environment to the SAUTE framework.
+        - SimmerAdapter: Adapt the environment to the SIMMER framework.
 
+        Args:
+            env_id: The environment id.
+            num_envs: The number of environments.
```
Comment (muchvo) [inline omnisafe/adapter/online_adapter.py:76]: how about considering type of parameters?

**Review by muchvo — APPROVED** [review]
LGTM.

**Pull request merged**